repositorytools/cli/common.py

```diff
@@ -55,5 +58,5 @@ def run(self, args=None):
         return args_namespace.func(args_namespace)
 
     def __call__(self, *args):
-        self.run(args)
+        self.run(*args)
```

Comment:
Merged in #30, but you can probably keep it here, because github doesn't complain about it ;)